src/fmi3Functions.c

```diff
@@ -666,7 +666,7 @@ fmi3Status fmi3GetBinary(fmi3Instance instance,
 
     for (size_t i = 0; i < nValueReferences; i++) {
         size_t index = 0;
-        Status s = getBinary(S, valueReferences[i], valueSizes, (const char**)values, &index);
+        Status s = getBinary(S, (ValueReference) valueReferences[i], valueSizes, (const char**)values, &index);
```

Comment:
I'd avoid the extra blank after the cast (like the other cast in the same line). (Also for all other changes.)

Reply:
fixed